src/js/components/Field/Field.stories.tsx

```diff
@@ -0,0 +1,208 @@
+  );
+  const disabled = boolean('Disabled', false);
+
+  return (
+    <>
+      <h4>
+        Captions can be added to explain how the entered data is used, why is
+        required, or similar.
+      </h4>
+      <br />
+      <Field
+        caption={caption}
+        isDisabled={disabled}
+        label="Primary Email Address"
+        leftIcon={<Icon name="home" />}
+        name="Sample"
+        placeholder={placeholder}
+        status={status}
+      />
+    </>
+  );
+});
+
+stories.add("Sizes", () => {
+  const status = select("status", statusOptions);
+  const placeholder = text("Placeholder", "Input Placeholder");
+  const disabled = boolean('Disabled', false);
+
+  return (
+    <>
+      <h4>
+        Inputs match their container's width, but height, font size and more can
+        be controlled via the Size property.
+      </h4>
+      <br />
+      <Field
+        caption="This input is small"
+        isDisabled={disabled}
+        label="Sizes.SMALL or 'small'"
+        leftIcon={<Icon name="home" />}
+        name="Sample"
+        placeholder={placeholder}
+        size={Sizes.SMALL}
+        status={status}
+      />
+      <Field
+        caption="This input is default"
+        isDisabled={disabled}
+        label="No size provided (default/medium)"
+        leftIcon={<Icon name="home" />}
+        name="Sample2"
+        placeholder={placeholder}
+        status={status}
+      />
+      <Field
+        caption="This input is large"
+        isDisabled={disabled}
+        label="Sizes.LARGE or 'large'"
+        leftIcon={<Icon name="home" />}
+        name="Sample3"
+        placeholder={placeholder}
+        size={Sizes.LARGE}
+        status={status}
+      />
+    </>
+  );
+});
+
+stories.add("Field with Icons", () => {
+  const status = select("status", statusOptions);
+  const placeholder = text("Placeholder", "Input Placeholder");
+  const size = select("size", sizeOptions);
+  const disabled = boolean('Disabled', false);
+
+  return (
+    <Field
+      caption="This input is..."
+      isDisabled={disabled}
+      label="This is a sample input"
+      leftIcon={<Icon name="home" />}
+      name="Sample"
+      placeholder={placeholder}
+      rightIcon={<Icon name="search" />}
+      size={size}
+      status={status}
+    />
+  );
+});
+
+stories.add("Status Icons", () => {
+  const status = select("status", statusOptions);
+  const placeholder = text("Placeholder", "Input Placeholder");
+  const size = select("size", sizeOptions);
+  const disabled = boolean('Disabled', false);
+
+  return (
+    <Field
+      caption="This input is..."
+      isDisabled={disabled}
+      label="This is a sample input"
+      leftIcon={<Icon name="home" />}
+      name="Sample"
+      placeholder={placeholder}
+      rightIcon={<Icon name="search" />}
+      showStatus
+      size={size}
+      status={status}
+    />
+  );
+});
+
+stories.add("Clear Icon", () => {
```

Comment:
Can you specify a clear icon or control what clears the input? It seems a bit strange to clear by clicking the status icon.

Reply:
Sorry David, we found that icon moments ago and I hadnt pushed it up yet. All of these icons will eventually be replaced by the new icons, so we're trying not to get too hung up on them at the moment.